neurolang/probabilistic/tests/test_probabilistic_solvers.py

```diff
@@ -707,7 +707,7 @@ def test_repeated_variable_probabilistic_rule(solver):
 
 
 def test_repeated_variable_with_constant_in_head(solver):
-    if solver is not dichotomy_theorem_based_solver:
+    if solver is not small_dichotomy_theorem_based_solver:
         pytest.skip()
```

Comment:
and here and every other one that is pertinent. Skipping the test really breeds for bugs